consensus/scp/src/scp_log.rs

```diff
@@ -357,48 +273,3 @@ impl<V: serde::de::DeserializeOwned + Value> Iterator for ScpLogReader<V> {
         Some(data)
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
We're no longer doing anything complicated with directories, we just `create_dir_all` so there's no value in tests tests.